tests/test_dicom.py

```diff
@@ -70,13 +70,11 @@ def test_stop_before_pixels(self, dicom_object, shape_override):
             UniformSample(4, method="count"),
         ],
     )
-    def test_volume_handling(self, dicom_object, handler, mocker):
-        np.random.seed(42)
-        F = 8
+    def test_volume_handling(self, dicom_object_3d, handler, mocker, transfer_syntax):
         spy = mocker.spy(handler, "__call__")
-        dicom_object.NumberOfFrames = F
-        dicom_object.PixelData = np.random.rand(F, 128, 128).tobytes()
-
-        array1 = read_dicom_image(dicom_object, volume_handler=spy)
+        F = 8
```

Comment:
See earlier comment---I think we should consider avoiding single capital letter variable names.